uiowa/uiowa
# Block List Grid Options

Resolves #2309
Relates to https://github.com/uiowa/uiowa/issues/2058

This introduces list/grid options for articles, events, people, rss feed. Editors can choose list or grid. If grid, they can choose 2 column or 3 column.

Depending on where the block is placed, the grid will work. Of course it will work in a single column desktop layout, but as the viewport changes and the columns become narrower the grid falls back to a list display similar to how individual cards stack currently.

# To Test

`blt frontend` and sync a v3 site (e.g. sandbox.uiowa.edu).
- New/existing blocks should have a list default, and if grid is selected, 3 column should be the default selection.
- If list is selected, the grid classes should not apply to the block render (e.g. grid--threecol--33-34-33).
- Try the different block options in different layouts with different percentage columns to make sure it is responsive enough that you don't cringe.
